internal/uvm/resourcepaths.go

```diff
@@ -0,0 +1,21 @@
+package uvm
+
+const (
+	gpuResourcePath                  string = "VirtualMachine/ComputeTopology/Gpu"

```

Comment:
what do we anticipate this will be used for? Just curious

Reply:
Some part of me said "only add what we use" but the other part of me said "this is what is possible in the platform and is not exposed". So we may never use gpu say but it is a resource that can be added. I can remove the ones we are not likely to ever use if you want.

Reply:
I'm fine with exposing everything that's available :)